infra/helper.py

```diff
@@ -268,7 +280,11 @@ def _add_sanitizer_args(
         parser,
         choices=('address', 'memory', 'undefined', 'coverage', 'dataflow')):
   """Add common sanitizer args."""
-  parser.add_argument('--sanitizer', default='address', choices=choices)
+  parser.add_argument(
+      '--sanitizer',
+      default=None,
+      choices=choices,
+      help='the default is "address"; "dataflow" for "dataflow" engine')
```

Comment:
Not sure we want to document this behavior for "dataflow", maybe remove the help string.

Reply:
Can change to `the default is "address", unless using "dataflow" engine`, does that sound better?